doc/source/data/api/input_output.rst

```diff
@@ -105,6 +105,14 @@ Arrow
 .. automethod:: ray.data.Dataset.to_arrow_refs
     :noindex:
 
+MongoDB
+-------
+
+.. autofunction:: ray.data.read_mongo
+
+.. autofunction:: ray.data.Dataset.write_mongo
+    :noindex:
+
```

Comment:
Also we need to add `MongoDatasource` at line 213 below.

Reply:
Done.